fe/fe-core/src/main/java/org/apache/doris/mysql/MysqlHandshakePacket.java

```diff
@@ -25,11 +25,12 @@ public class MysqlHandshakePacket extends MysqlPacket {
     // JDBC uses this version to check which protocol the server support
     // Set the patch version to 99 to prevent the vulnerability scanning tool from
     // falsely reporting MySQL vulnerabilities
-    public static final String SERVER_VERSION = "5.7.99";
+    public static final String SERVER_VERSION = "8.0.31";
```

Comment:
This version may not be updated due to TLS does not complete all the features of 8.0.31.